### PR TITLE
Prototype qdrant & Streamlit creation

### PR DESCRIPTION
**Pull request by evesonr**
Creating a qdrant collection and "straw man" dashboard, containerised. The dashboard returns feedback records as they appear in BigQuery, allowing a user to provide a single keyword to search over the collection. The user can also provide a list of subject page paths to filter on, can select the number of results they want to return (k) and can filter on datetime.

To test, create a `docker-compose.env `file with the relevant env vars, and run `docker-compose up`

### CODE REVIEW

File: src/collection/query_collection.py
Comment (hwrightson) [inline src/collection/query_collection.py:30]: Should this be `limit = k`?

File: src/collection/query_collection.py
Comment (hwrightson) [inline src/collection/query_collection.py:34]: Same as above

File: src/sql_queries.py
Comment (hwrightson) [inline src/sql_queries.py:5]: Would it be easier to change this to DATE(feedback.created) so you don't have to adjust for datetime in other files? 

Same with CAST(feedback.feedback_record_id AS STRING)?

**Review by hwrightson — COMMENTED** [review]
Added some comments without explicit approval, please take a look and we can discuss, then I can either approve or you can make the additional streamlit design changes and I add them and I can approve all at once.

File: app/main.py
Comment (evesonr) [reply app/main.py:20]: Yes and yes. This depends on how we want to create embeddings for feedback, and as we already had SentenceTransformer embeddings from HF stored in BigQuery it made sense to use HF here. Using the Inference API would be preferable but costs, so I opted to download the relatively small mpnet model on startup instead, as only takes a few seconds.

In "prod", we might opt for a local model like this, or Inference API, or OpenAI embeddings.

File: src/collection/query_collection.py
Comment (evesonr) [reply src/collection/query_collection.py:30]: Yes, I spotted this last night after our session. Will push this change this morning

File: src/sql_queries.py
Comment (evesonr) [reply src/sql_queries.py:5]: Agree, will change

File: collection/create_collection.py
Comment (evesonr) [reply collection/create_collection.py:1]: Agree, and yes go for it!

File: .gitignore
Comment (evesonr) [reply .gitignore:5]: Interesting... it doesn't show up in this PR? I think it gets created by our pre-commit hook, so it may have been created locally when you last made changes

**Review by hwrightson — APPROVED** [review]
LGTM

**Pull request merged**
